src/main/handlers.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+func index(c echo.Context) error {
```

Comment:
This should be indexHandler() and the other ones should look similar. Like getDrivesHandler(). Can you change it before I merge it?